tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist = py{36,37,38,39},fuzz
+
+[testenv]
+setenv = PYTHONPATH = {toxinidir}/src
+skip_install = True
+deps =
+    -r{toxinidir}/test_requirements.txt
+commands =
+    pip install -e .[d]
+    coverage erase
+    coverage run -m pytest tests
+    coverage report
+
+[testenv:fuzz]
+skip_install = True
+deps =
+    -r{toxinidir}/test_requirements.txt
+    hypothesmith
+    lark-parser < 0.10.0
+; lark-parser's version is set due to a bug in hypothesis. Once it solved, that would be fixed.
```

Comment:
@saroad2 It looks like `lark-parser`'s been updated in the [test dependencies for `hypothesmith`](https://github.com/Zac-HD/hypothesmith/blob/d34c2947e20385c2d1a337b337027d6d39ada3af/deps/test.txt#L35-L36) - could/should we remove this version pin?

Reply:
I guess we should :) I added this pinning so the tests would pass.